bootstrap.zsh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env zsh
+
+set -euo pipefail
+
+echo "Resetting the project's virtual environment"
+rm -rf .venv
+python -m venv .venv
+
+echo 'Installing packages'
+. .venv/bin/activate
+poetry install
```

Comment:
This assumes that poetry is already installed in the current python environment, correct? Maybe it would be useful to mention that in the README - that this package will not install poetry for you, and assumes that poetry is already installed.

Reply:
Added in 148135f